### PR TITLE
Created ability to upload and download files to Zanata

### PR DESCRIPTION
**Pull request by chalettu**
PV https://www.pivotaltracker.com/story/show/144100935

### CODE REVIEW

**Comment by chalettu** [comment]
@miq-bot add_label fine/no
@miq-bot add_label enhancement

**Comment by chalettu** [comment]
@AllenBW do you want to take another look now that I added yarn tasks for download and upload

File: package.json [hunk 1]
```diff
@@ -109,12 +111,14 @@
   "devDependencies": {
     "@manageiq/manageiq-api-mock": "1.2.3",
     "angular-mocks": "1.6.4",
+    "axios": "^0.16.1",
```
Comment (AllenBW) [inline package.json]: Need to lock this into a specific version (kill da carrot)

Comment (AllenBW) [inline package.json]: yeah just this one thing, rest feels gooooooood

**Pull request merged**